app/models/qing_group_divided.rb

```diff
@@ -0,0 +1,13 @@
+class QingGroupDivided
```

Comment:
Can we call this MockQingGroup and put a comment on the first line explaining its purpose. Otherwise it's hard to understand.